### DISCUSSION TITLE
Stop metadata being written into LoRA files?

### DISCUSSION THREAD

**Discussion by shadowlocked** [bmaltais/kohya_ss#1500] [Q&A]
Is it possible to stop details about the annotations being included as metadata? As far as I can tell, it cannot be edited out without invalidating the LoRA so that it won't be read.

**EDIT:** Anyone who does still want to create a LoRA file without metadata, they can train as ckpt instead of safetensors and use the AUTOMATIC1111 checkpoint merger to get a 1>1 conversion to zero-metadata safetensors format (will reproduce identical images from same seed). Details [here](https://old.reddit.com/r/StableDiffusion/comments/16d4s7n/editing_or_removing_metadata_from_a_lora_file/).

**Comment by TeKett** [comment]
Annotation?

Regarding the topic. May i ask why you don't want metadata? Since it don't really matter, and its a good thing if you share it, since people can see the checkpoint, tokens, clip skip, image size, type of network, some other settings, etc, conveniently in the GUI rather then you having to write it into the description yourself, and they having to manage the info somewhere.

**Reply by shadowlocked** [reply]
I think it's reasonable to gain control over metadata early, while it doesn't matter much. Things like this, and the currently unused IP flag, are placeholders for potentially undesired functionality down the road. So in a system this granular, in terms of control, it is pretty crazy, I think, that we cannot control this aspect too.

Additionally, someone else on the SD discord needed to hand over a LoRA to a client, and did not want the client seeing the metadata annotations, since they were useful, for the efficiency of the model, but perhaps rather painful if the client had seen them himself. 

So I'm not a big fan of the 'Nothing to hide, nothing to fear' mentality in general, and can see no reason not to make this functionality accessible to users, and not worry about their reasons.

**Comment by sbuldan** [comment]
I have exact opposite of this problem, it only shows my lora folder name in training dataset tags. Does anyone know how may I fix it?

**Comment by Yinzo** [comment]
Here has a web tool for editing LoRA metadata, incase you still need it: [https://lorainfo.tools/](https://lorainfo.tools/)

**Comment by Syscore64** [comment]
why not use --no_metadata?
